app/annotations.py

```diff
@@ -1,10 +1,45 @@
 from functools import wraps
 
-# from flask import current_app
 from inspect import signature
 
-from app import signer_notification
-from app.encryption import SignedNotification, SignedNotifications
+
+def sign_param(func):
```

Comment:
This seems like a great idea but I can't tell if it works just by looking at the code. Could you add some instructions for how to test if this is working?